core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/tenant/JDBCTenantManager.java

```diff
@@ -871,6 +871,55 @@ public void deleteTenant(int tenantId) throws UserStoreException {
         }
     }
 
+    public void deleteTenant(String tenantUniqueID) throws UserStoreException {
```

Comment:
Java doc missing

Reply:
Interface having the Java doc comment thatswhy didn't add it.